### DISCUSSION TITLE
How to use GraphAr with GraphScope

### DISCUSSION THREAD

**Discussion by kesavkolla** [apache/incubator-graphar#120] [Q&A] [answered]
Does these projects work together? I am planning to use spark to compute graphs for my existing data. My understanding is once graph is computed using GraphScope we can persist data in GraphAr file format. Can someone show some light on these two projects?

**Comment by acezen** [comment, marked as answer]
The GraphAr is a system-independent file format of graph, it's not binding to  any graph system。GraphScope is our first landing system that support building graph from GraphAr file format and writing graph to GraphAr (as an application of GraphAr). This feature of GraphScope would be release in v0.20.0.
